descriptions/api.github.com/api.github.com.yaml

```diff
@@ -4021,8 +4021,8 @@ paths:
         description: "Indicates which sorts of issues to return. Can be one of:  \n\\*
           `assigned`: Issues assigned to you  \n\\* `created`: Issues created by you
           \ \n\\* `mentioned`: Issues mentioning you  \n\\* `subscribed`: Issues you're
-          subscribed to updates for  \n\\* `all`: All issues the authenticated user
-          can see, regardless of participation or creation"
+          subscribed to updates for  \n\\* `all` or `repos`: All issues the authenticated
+          user can see, regardless of participation or creation"
```

Comment:
I don't quite understand, is `repos` an alias for `all`? Is there any difference between the two? Is `repos` deprecated? If so, I would not mention it in the public spec.

Reply:
ping @mattpage ☝🏼

Reply:
update: it is an alias